### PR TITLE
Implement #524 Allow flat colors. Adds an edit control for Ambient li…

### PR DESCRIPTION
**Pull request by phkahler**
@whitequark  can you have a look at the text window code here? The change[c] link isn't working.

When that's fixed we should be able to turn off the 2 directional lights (set intensity to 0) and turn up ambient to 1 to get flat colors.

### CODE REVIEW

**Comment by phkahler** [comment]
@whitequark never mind, got it.  It seems to work. Setting both lights brightness to zero and the ambient to 1.0 yields flat coloring.

File: src/confscreen.cpp [hunk 1]
```diff
@@ -218,6 +224,10 @@ void TextWindow::ShowConfiguration() {
             CO(SS.lightDir[i]), i, &ScreenChangeLightDirection,
             SS.lightIntensity[i], i, &ScreenChangeLightIntensity);
     }
+    Printf(false, "%Bp         Ambient Lighting     "
```
Comment (whitequark) [inline src/confscreen.cpp:227]: Actually, this should be lowercase to follow the style of that dialog.
```suggestion
    Printf(false, "%Bp         ambient lighting     "
```

Comment (phkahler) [reply src/confscreen.cpp]: Fixed. Good call it's all lower case.

**Comment by whitequark** [comment]
Thanks!

**Pull request merged**